src/java.desktop/unix/native/libawt_xawt/awt/gtk3_interface.c

```diff
@@ -2406,6 +2406,9 @@ static gint gtk3_get_color_for_state(JNIEnv *env, WidgetType widget_type,
         } else if (widget_type == MENU_BAR && state_type == GTK_STATE_INSENSITIVE
             && color_type == FOREGROUND) {
             widget_type = MENU;
+        } else if (widget_type == PASSWORD_FIELD && state_type == GTK_STATE_INSENSITIVE
```

Comment:
Looks like this line and a few others nearby exceed the character limit per line if you wanted to correct that as well.

Reply:
Updated the character limit for the current fix. I think other lines can be done later.